### DISCUSSION TITLE
Model auto stop?

### DISCUSSION THREAD

**Discussion by mihail342** [ggml-org/llama.cpp#207] [Q&A] [answered]
Hello, I'm new to this business. Tell me, is it possible to cancel the shutdown of the model after a few questions in interactive mode? Is it possible to keep the model running for a long time after launch?
the model stops automatically with these lines

main: mem per token = 22439492 bytes
main:     load time =  4926.95 ms
main:   sample time =   115.04 ms
main:  predict time = 137328.81 ms / 353.94 ms per token
main:    total time = 227331.20 ms

Is it possible to make changes to the code to increase the model's running time?

**Comment by bitRAKE** [comment, marked as answer]
Sometimes I don't get the `[end of text]` message - not sure why, yet.

#181 PR attempts to address this by ignoring the end code sent by the model. Maybe there should be other filtering of the characters as the current code will output control characters (think backspace, bell, ...) - probably not desirable.

**Reply by mihail342** [reply]
thanks for the answer
